packages/webapp/src/utils/acounts.ts

```diff
@@ -0,0 +1,120 @@
+/**
+ * Utilities to deal with burner accounts.
+ *
+ * @module utils/accounts
+ */
+
+import {
+    createPublicClient,
+    createWalletClient,
+    http,
+    getContract,
+    concat,
+    encodeFunctionData,
+} from 'viem'
+import { localhost } from 'viem/chains'
+import { generatePrivateKey, privateKeyToAccount } from 'viem/accounts'
+import {
+    simpleAccountFactoryAbi,
+    simpleAccountAbi,
+    paymasterAbi
+} from 'src/utils/abis'
+
+interface UserOperation {
+    sender: `0x${string}`
+    nonce: number
+    initCode: `0x${string}`
+    callData: `0x${string}`
+    callGasLimit: number
+    verificationGasLimit: number
+    preVerificationGas: number
+    maxFeePerGas: number
+    maxPriorityFeePerGas: number
+    paymasterAndData: `0x${string}`
+    signature: `0x${string}`
+}
+
+const publicClient = createPublicClient({
+    chain: localhost,
+    transport: http()
+})
+
+const paymasterAddress = process.env.PAYMASTER_ADDRESS as `0x${string}`
+const paymaster = getContract({
+    address: paymasterAddress,
+    abi: paymasterAbi,
+    publicClient
+})
+
+export async function createAccount() {
+
+    // use simple account factory to deploy a new smart account
+    // get the address of the smart account
+    // create sdk that reads private key from the browser storage and signs transactions
+
+    const privateKey = generatePrivateKey()
+    // store this private key in browser storage (is this secure?)
+    localStorage.setItem("privateKey", privateKey)
```

Comment:
This should be probably added as an atom in the store, using `atomWithStorage` (just like we use for the gameID).

I don't think putting it in the store will give us a ton of benefits, but at least it keeps all the state (and all the persistent state) in one place, making it a little bit easier to reason about / explore the codebase.